var/spack/repos/builtin/packages/py-pydmd/package.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPydmd(PythonPackage):
+    """PyDMD is a Python package that uses Dynamic Mode Decomposition
+    for a data-driven model simplification based on spatiotemporal
+    coherent structures."""
+
+    homepage = "https://mathlab.github.io/PyDMD/"
+    url      = "https://github.com/mathLab/PyDMD/archive/v0.3.tar.gz"
+
+    import_modules = ['pydmd']
```

Comment:
Now that #20023 has been merged, `import_modules` should be auto-detected. You may not need this line anymore.